ui/app/rif/components/table/genericTable.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { useTable, usePagination } from 'react-table';
+import Pagination from './pagination';
+
+const Table = ({ title, columns, data, pageSize, displayColumnHeader, classes }) => {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    prepareRow,
+    page,
+    canPreviousPage,
+    canNextPage,
+    pageOptions,
+    gotoPage,
+    nextPage,
+    previousPage,
+    state: { pageIndex },
+  } = useTable(
+    {
+      columns,
+      data,
+      initialState: { pageIndex: 0, pageSize: pageSize || 10 },
+    },
+    usePagination,
+  );
+  const styles = classes || {};
+  return (
+    <>
```

Comment:
What's this <> can you explain?

Reply:
This is just to wrap something, is the same as an empty <div>

Reply:
There i changed it to be <div> instead of <>